### PR TITLE
Adds Windows CI, and gets tests passing on Windows

### PR DESCRIPTION
**Pull request by timhaines**

### CODE REVIEW

File: lib/percy/client/environment.rb [hunk 1]
```diff
@@ -90,7 +90,11 @@ def self._commit_sha
       # @private
       def self._raw_commit_output(commit_sha)
         format = GIT_FORMAT_LINES.join('%n') # "git show" format uses %n for newlines.
-        output = `git show --quiet #{commit_sha} --format="#{format}" 2> /dev/null`.strip
+        if Gem.win_platform?
```
Comment (timhaines) [inline lib/percy/client/environment.rb]: `Gem.win_platform?` is supported in MRI 1.9 onwards.

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+environment:
+  matrix:
+    - RUBY_VERSION: 22
```
Comment (djones) [inline appveyor.yml]: Worth nothing this means Ruby 2.2.6p396

File: appveyor.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    - RUBY_VERSION: 21
```
Comment (djones) [inline appveyor.yml]: And this means Ruby 2.1.9p490

File: appveyor.yml (hunk 2 above)
Comment (timhaines) [reply appveyor.yml]: At time of writing, yeah.  I suspect Appveyor will update the versions, but keep 22 meaning Ruby 2.2.xxx and 21 meaning Ruby 2.1.xxx

**Pull request merged**